docs/pages/management/operations/backup-restore.mdx

```diff
@@ -137,6 +137,15 @@ $ sudo teleport start --config fresh.yaml --bootstrap state.yaml
 $ tctl --config fresh.yaml get all
 # <your state here>
 ```
+<Notice>
+The `tctl get all` is not fully supported remotely as retrieving user
+resources with secrets is not allowed. As a workaround, retrieve resources
+individually. Local user secrets including passwords would need resetting
+after creating without secrets.
+
+All trusted cluster certificate resources should be removed for use
```

Comment:
Similarly, do we think the reader will know what to do here? This is clear to me, but I suspect it will still be confusing for newcomers.

Reply:
Please see update @zmb3